src/auth/strategies/discord.strategy.ts

```diff
@@ -0,0 +1,42 @@
+import { Inject, Injectable } from "@nestjs/common";
+import { PassportStrategy } from "@nestjs/passport";
+import { Strategy } from "passport-discord";
+import { Profile } from "passport-discord";
```

Comment:
```suggestion
import { Strategy, Profile } from "passport-discord";
```
you can combine the imports. unless you wanted to import the `profile` types from "passport-discord"

Reply:
thanks, that's some weird auto import 😂